docs/TuxOwnerClass.md

```diff
@@ -0,0 +1,307 @@
+
+If you are unfamiliar with React components please review their [docs](http://facebook.github.io/react/). Below is the API documentation for the `createOwnerClass` method.
+
+***
+
+### <a id="React-createOwnerClass"></a>3) React.createOwnerClass [#](#React-createOwnerClass)
+Returns an `OwnerClass` based on the passed in `ownerClassProps`
+
+```javascript
+    var OwnerClass = React.createOwnerClass(ownerClassProps);
+```
+
+#### <a id="createOwnerClass-ownerClassProps"></a>Parameter - `ownerClassProps` - type: OBJECT - required [#](#createOwnerClass-ownerClassProps)
+Properties of the `OwnerClass` to be created. Expected keys:
+
+##### <a id="ownerClassProps-render"></a>Property - `ownerClassProps.render` - type: FUNCTION - required [#](#ownerClassProps-render)
+As with any other `React` class, a [render](#http://facebook.github.io/react/docs/component-specs.html#render) method is required. This method is responsible for returning a `React Element` which can be rendered onto the DOM.
+
+```
+    ownerClassProps.render = function () {
+      return (
+        <div>
+          <h1>hello world</h1>
+        </div>
+      );
+    };
+```
+
+##### <a id="ownerClassProps-LIFECYCLE_AND_SPECS"></a>Methods/Properties - `ownerClassProps.LIFECYCLE_AND_SPECS` - type: FUNCTIONs - optional [#](#ownerClassProps-LIFECYCLE_AND_SPECS)
+As with any other `React` class, the `ownerClassProps` can optionally include `React` [lifecycle methods and specs](#http://facebook.github.io/react/docs/component-specs.html).
+
+```
+    ownerClassProps.componentDidMount = function () {
+      roomActions.get();
+    };
+```
+
+***
+
+##### <a id="ownerClassProps-connectOwnerToStore"></a>3.1) Method - `ownerClassProps.connectOwnerToStore` - type: FUNCTION or ARRAY of FUNCTIONs - optional [#](#ownerClassProps-connectOwnerToStore)
+This method allows an `OwnerClass` to easily and semantically add and remove event listeners for stores. The listeners will be added on `componentDidMount` and will be removed on `componentWillUnmount`. The `connectOwnerToStore` method must return an object which the `OwnerClass` will use to connect itself to the store. Alternatively, `connectOwnerToStore` can be an array of functions that return objects if the `OwnerClass` needs to connect to multiple stores. In either case, the function(s) will be invoked with the context of the `Owner Component` just as with any other `React` spec or lifecycle method.
+
+```
+    ownerClassProps.connectOwnerToStore = function () {
+      //'this' will be the Owner Component so we can invoke methods like this.setState
+      return connectOwnerToStoreProps;
+    };
+
+    connectOwnerToStoreProps = {
+      store: roomStore,
+      listener: [ownerClassProps.onRoomStoreChange, ownerClassProps.onRoomStoreSpecialChange],
```

Comment:
Do the listener and events map one to one? If there are more of one or vice versa what happens? Might be good to add a note for those situations.